tests/NuGet.Services.AzureSearch.FunctionalTests/Config/Prod.json

```diff
@@ -1,12 +1,14 @@
 ﻿{
```

Comment:
Should we even have these configs here? IMO these should be private.

Reply:
no idea why these were added in the first place, @loic-sharma - fine to get rid of all these config files and simply have one sample "config.json"/"dev.json" which can define the config parameters for easy local run/debug?

Reply:
deleting int and prod.json configs. Keeping the Dev.json for local debugging since its used as the default configuration name.

Reply:
Yes, these are fine to delete.